src/index.js

```diff
@@ -1,20 +1,78 @@
 import './style.css';
 
-const myTodo = [
-  { description: 'Going to the GYM', completed: true, index: 1 },
-  { description: 'Cleaning my Room', completed: false, index: 2 },
-  { description: 'Washing my Car', completed: true, index: 3 },
-  { description: 'Coding for 4 hours', completed: false, index: 4 },
-];
-const list = document.getElementById('list');
-// Sorting the array first
-myTodo.sort((a, b) => a.index - b.index);
-
-myTodo.forEach((value) => {
-  // Creating list of to-do
-  const li = document.createElement('li');
-  li.innerHTML = `
-        ${value.description}<i class="fa-solid fa-ellipsis-vertical"></i>
+const save = (data) => {
+  localStorage.setItem('todo', JSON.stringify(data));
+};
+const retrieve = () => JSON.parse(localStorage.getItem('todo'));
+
+let display;
+const localData = retrieve();
+if (!localData) localStorage.setItem('todo', '[]');
+
+const updateList = (todos) => todos.map((value, index) => {
+  value.id = index;
+  return value;
+});
+
+const add = (description, completed, id) => {
+  const storeData = retrieve();
+  storeData.push({ description, completed, id });
+  const sortedData = updateList(storeData);
+  save(sortedData);
+};
+
+const remove = (id) => {
+  const storeData = retrieve();
+  const remaining = storeData.filter((todo) => todo.id !== id);
+  const sortedData = updateList(remaining);
+  save(sortedData);
+  display();
+};
```

Comment:
- [ ] You did great adding the `remove and add` functionality. :+1: :+1: :100: Your project works fine, but there is one crucial requirement that you missed and you need to implement this before moving to the next milestone. :+1: 

> Create a new JavaScript file for the new functionality.

- - According to this requirement, you are meant to create a new file and implement the `add and remove` functionality in that file and then you can import the functions inside your index file. This encourages code readability and reusability and makes your code easy to maintain. I'll suggest you go through your code base and implement this requirement. :smile: :+1: